src/ST-Player/PlayerStats.cs

```diff
@@ -47,6 +47,37 @@ public void Reset()
     }
 }
 
+public class CheckpointObject
+{
+    public int CP { get; set; }
+    public int RunTime { get; set; } // To-do: what type of value we use here? DB uses DECIMAL but `.Tick` is int???
+    public int Ticks { get; set; } // To-do: this was supposed to be the ticks but that is used for run_time for HUD????
```

Comment:
Ticks = run time, so we only need one variable for it, and that variable is an integer. I recommend using `Ticks` wherever instead of Runtime, as Runtime should probably refer to Ticks / Tickrate for the actual time value in seconds

Reply:
Except for maybe `Map.cs` all should be like this, even if not named `Ticks` the type is an `integer` for this everywhere 🤔